code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -712,6 +712,7 @@ GLOBAL_VAR(bomb_set)
 	if(!training)
 		GLOB.poi_list |= src
 		GLOB.nad_list |= src
+		GLOB.high_risk_items |= UID()
```

Comment:
We already use a global list for high value items from the antag menu. Please use that instead.